%%
%% Copyright 2022 OXFORD UNIVERSITY PRESS
%%
%% This file is part of the 'oup-authoring-template Bundle'.
%% ---------------------------------------------
%%
%% It may be distributed under the conditions of the LaTeX Project Public
%% License, either version 1.2 of this license or (at your option) any
%% later version.  The latest version of this license is in
%%    http://www.latex-project.org/lppl.txt
%% and version 1.2 or later is part of all distributions of LaTeX
%% version 1999/12/01 or later.
%%
%% The list of all files belonging to the 'oup-authoring-template Bundle' is
%% given in the file `manifest.txt'.
%%
%% Template article for OXFORD UNIVERSITY PRESS's document class `oup-authoring-template'
%% with bibliographic references
%%

%%%CONTEMPORARY%%%
\documentclass[unnumsec,webpdf,contemporary,large]{oup-authoring-template}%
%\documentclass[unnumsec,webpdf,contemporary,large,namedate]{oup-authoring-template}% uncomment this line for author year citations and comment the above
%\documentclass[unnumsec,webpdf,contemporary,medium]{oup-authoring-template}
%\documentclass[unnumsec,webpdf,contemporary,small]{oup-authoring-template}

%%%MODERN%%%
%\documentclass[unnumsec,webpdf,modern,large]{oup-authoring-template}
%\documentclass[unnumsec,webpdf,modern,large,namedate]{oup-authoring-template}% uncomment this line for author year citations and comment the above
%\documentclass[unnumsec,webpdf,modern,medium]{oup-authoring-template}
%\documentclass[unnumsec,webpdf,modern,small]{oup-authoring-template}

%%%TRADITIONAL%%%
%\documentclass[unnumsec,webpdf,traditional,large]{oup-authoring-template}
%\documentclass[unnumsec,webpdf,traditional,large,namedate]{oup-authoring-template}% uncomment this line for author year citations and comment the above
%\documentclass[unnumsec,namedate,webpdf,traditional,medium]{oup-authoring-template}
%\documentclass[namedate,webpdf,traditional,small]{oup-authoring-template}

%\onecolumn % for one column layouts

%\usepackage{showframe}
\usepackage[T1]{fontenc}
\graphicspath{{Fig/}}

% line numbers
%\usepackage[mathlines, switch]{lineno}
%\usepackage[right]{lineno}

\theoremstyle{thmstyleone}%
\newtheorem{theorem}{Theorem}%  meant for continuous numbers
%%\newtheorem{theorem}{Theorem}[section]% meant for sectionwise numbers
%% optional argument [theorem] produces theorem numbering sequence instead of independent numbers for Proposition
\newtheorem{proposition}[theorem]{Proposition}%
%%\newtheorem{proposition}{Proposition}% to get separate numbers for theorem and proposition etc.
\theoremstyle{thmstyletwo}%
\newtheorem{example}{Example}%
\newtheorem{remark}{Remark}%
\theoremstyle{thmstylethree}%
\newtheorem{definition}{Definition}

\begin{document}

\journaltitle{The ISME Journal}
\DOI{DOI HERE}
\copyrightyear{2024}
\pubyear{2024}
\access{Advance Access Publication Date: Day Month Year}
\appnotes{Paper}

\firstpage{1}

%\subtitle{Subject Section}

\title[Crete's soil microbiome]{Exploring the Soil Microbiome coupling with the Crete island Data Cube}

\author[1,2]{Savvas Paragkamian\ORCID{0000-0002-8508-2521}}
\author[3,4]{Johanna B. Holm\ORCID{0000-0002-7646-4085}}
\author[5]{Christina Pavloudi\ORCID{0000-0001-5106-6067}}
\author[6]{Haris Zafeiropoulos\ORCID{0000-0002-4405-6802}}
\author[1]{Christos A. Christakis\ORCID{0000-0002-7075-0996}}
\author[1]{Melanthia Stavroulaki\ORCID{0000-0003-4392-7159}}
\author[1]{Dimitris Tsaparis\ORCID{0000-0001-9626-5553}}
%\author[1]{Iraklis Vretzakis\ORCID{0000-0001-9626-5553}}
\author[7]{Stephanie A. Yarwood\ORCID{0000-0003-4593-7908}}
\author[2]{Antonios Magoulas}
\author[1]{Panagiotis F. Sarris\ORCID{0000-0001-7000-8997}}
\author[2]{Giorgos Kotoulas\ORCID{0000-0002-4667-8533}}
\author[3]{Lynn Schriml\ORCID{0000-0001-8910-9851}}
\author[2,$\ast$]{Evangelos Pafilis\ORCID{0000-0001-5079-0125}}

\authormark{Paragkamian et al.}

\address[1]{\orgdiv{Department of Biology}, \orgname{University of Crete}, \orgaddress{\street{Voutes}, \postcode{70013}, \state{Crete}, \country{Greece}}}
\address[2]{\orgdiv{Institute of Marine Biology, Biotechnology and Aquaculture (IMBBC)}, \orgname{Hellenic Centre for Marine Research (HCMR)}, \orgaddress{\street{Gournes}, \postcode{71500}, \state{Crete}, \country{Greece}}}
\address[3]{\orgdiv{Institute for Genome Sciences}, \orgname{University of Maryland School of Medicine}, \orgaddress{\state{Baltimore}, \country{USA}}}
\address[4]{\orgdiv{Department of Microbiology and Immunology}, \orgname{University of Maryland School of Medicine}, \orgaddress{\state{Baltimore}, \country{USA}}}
\address[5]{\orgname{European Marine Biological Resource Centre}, \orgaddress{\state{Paris}, \country{France}}}
\address[6]{\orgdiv{Department of Microbiology, Immunology and Transplantation}, \orgname{Rega Institute for Medical Research, KU Leuven}, \orgaddress{\state{Leuven}, \country{Belgium}}}
\address[7]{\orgdiv{Department of Environmental Science and Technology}, \orgname{University of Maryland}, \orgaddress{\state{Baltimore}, \country{USA}}}

\corresp[$\ast$]{Corresponding author. \href{email:pafilis@hcmr.gr}{pafilis@hcmr.gr}}

\received{Date}{0}{Year}
\revised{Date}{0}{Year}
\accepted{Date}{0}{Year}

%\editor{Associate Editor: Name}

%\abstract{
%\textbf{Motivation:} .\\
%\textbf{Results:} .\\
%\textbf{Availability:} .\\
%\textbf{Contact:} \href{name@email.com}{name@email.com}\\
%\textbf{Supplementary information:} Supplementary data are available at \textit{Journal Name}
%online.}

\abstract{
    Microbes are known for their versatility, abundance
    and influence on soil ecosystem functioning.
    A synthesized knowledge base of microbial biodiversity, in terms of
    ecological and remote-sensing data remains a major challenge.
    Many worldwide studies have been published regarding soil
    microbiome ecosystems, though there are still many blind spots.
    Islands can be important case studies for this integration for more resolute and dense samplings.
    Here, we utilize the Island Sampling Day Crete 2016 microbial 16S rRNA gene
    amplicon data, integrated with soil and remote
    sensing data, to decipher the drivers of ecosystem function of the island.
    The Island Sampling Day Crete 2016 project has collected 144 topsoil samples
    from 72 sites, capturing a lot of this diversity, accompanied by FAIR
    (Findable, Accessible, Interoperable and Reproducible) data by design. 
    Cretan macroecology has been studied for centuries for its diverse  and endemic
    fauna and flora.
    In addition, Crete has been considered as a miniature continent with high contrasts in
    vegetation cover, elevation, climatic conditions. 
    We show that, higher altitudes in Crete found to
    be inhabited by a more diverse number of microorganisms, a pattern commonly
    seen in several faunistic groups, such as arthropods.
    The integration of the spatial data with state of the art methods enabled warning signals
    in pristine and grazing ecosystems.
    These results along with the
    climatic and desertification index influences on the soil microbiome of Crete,
    provide the basis to identify major drivers of biodiversity, to evaluate hotspots
    and contribute to foreknowledge of threatened ecosystems.
} 

\keywords{soil microbiome, networks, island biogeography, Crete, spatial data, remote sensing, data integration, FAIR data}

\maketitle

\section{Introduction}\label{intro}

Soil ecosystems are the cornerstone of terrestrial habitats, biodiversity and henceforth human activities.
Soils are characterised by multiple properties; chemical, physical and biological that 
form complex interdependant interactions. Biodiversity of soils covers
all forms of life, fauna, flora, bacteria, archaea, fungi, viruses. 
Bacteria and archaea are considered major drivers for the functionality of soil.
They infuence and are infuenced by their environment and their community structure 
defies their macroscopic functionality \cite{Bahram2018}.
Global soil microbiome studies have been employed to decipher soil microbiome
compositions \cite{thompson2017a-communal, Delgado-Baquerizo2018, Labouyrie2023},
functions \cite{Bahram2018} and biogeography \cite{Martiny2006, Guerra2020}.
These results showed the remarkable diversity in soils yet there are blind spots \cite{Guerra2020}
and these sampling are sparce when considering samples per area density. One of most resolute
study is by \cite{Karimi2020} which exemplified the 
vast complexity of soil bacterial communuties and the requirement of
dense samplings and isolated systems \cite{Dini-Andreote2021}.

Islands are nature's labs \cite{Whittaker2017} because of their isolation and smaller scale.
Borrowing this paradigm, soil microbes \cite{Li2020} and mycorrhizal fungi \cite{Delavaux2021} studies
in islands show the benefits of using islands as models. Ultimately islands can
set the ground to represent ecosystems with data and complex interactions \cite{Davies2016}.
This was also the case for the Island Sampling Day (ISD) project \cite{holm2024}
of the Genome Standards Consortium \cite{Field2011}
during the 18th workshop in June 2026 in Crete island, Greece. The goal was to "put standards into action"
in a soil microbiome survey with a dense sampling, 0.017 samples per km\textsuperscript{2}.
Hence, ISD is a large scale study in the confined space of the island of Crete which 
is considered a miniature continent \cite{Vogiatzakis2008_crete}.

Crete is a continental forarc island \cite{ali2016}, fifth largest island of the mediterranean (8350 km\textsuperscript{2}),
and a mediterranean biodiversity hotspot \cite{myers2000biodiversity}.
The island of Crete has been studied since the classical times for its'
fauna \cite{Sidiropoulos_Polymeni_Legakis_2017,Anastasiou2018Tenebrionid}, flora \cite{Krimbas_2005} and ecosystems \cite{Grove1993}.
Crete is home to the only endemic mammal of Greece, the Cretan shrew (\textit{Crocidura zimmermanni}),
more than 350 endemic arthropods \cite{bolanakis2024} and 183 endemic plants \cite{Kougioumoutzis2020}
among them a tree \textit{Zelkova abelicea}. Multifacet factors have shaped the
biodiversity of the island, for example the sharp elevation gradient \cite{trigas2013, FAZAN2017},
the complex evolutionary history \cite{POULAKAKIS2002} and the human - nature
interactions over thousands of years \cite{Vogiatzakis2008_med, Sfenthourakis2017}.
The major threats of human activities are becoming apperend in the island's ecosystems,
like desertification \cite{KARAMESOUTI2018266}, intesive grasing \cite{JouffroyBapicot2016},
climate change \cite{Kougioumoutzis2020,Vogiatzakis2016} and habitat loss \cite{ISPIKOUDIS1993259}.
Yet the topsoil microbial diversity of Crete has been unexplored.

Worldwide projects of microbiome studies have collected one or two topsoil
samples from Crete \cite{Vasar2022, Labouyrie2023, Bahram2018, Orgiazzi2018}.
Some have focused on soil fungi \cite{Mikryukov2023, Davison2021, Tedersoo2021}
and other soil eukaryotes \cite{Aslani2022}.
The only thorough microbiome study of a soil ecosystem in Crete, to our knowledge,
is in the north west part of the island, the Koiliaris Critical Zone Observatory \cite{tsiknia2014}.
Apart from the soil microbiome, soil physical and chemical properties has been
investigaded by globan and european projects like the Forum of European Geological Surveys
(FOREGS) \cite{nerc19017}, the Geochemical Mapping of Agricultural and Grazing Land
Soil in Europe (GEMAS) \cite{REIMANN2018302} and the Soil Profile Analytical
Database for Europe (SPADE) \cite{Hiederer2006}.

Data and metadata of these samplings are stored in different databases, yet 
great effort from these distinct communities have led to establishing standards
to enable FAIR data \cite{Wilkinson2016}. For amplicon sequences the Genome Standards
Consortium \cite{Field2011} has established the MIMARKS \cite{yilmaz2011minimum}
standards among others, and has been a advocate for open and unrestricted data \cite{Amann2019}.
Examples of rich metadata and platforms of hosting open data and digital soil maps are the 
European Soil Data Centre \cite{Panagos2022} and the World Soil Information
Service (WoSIS) of the ISRIC \cite{Batjes2024}. Apart from samplings,
spatial data are openly available terrestrial ecosystems.
Climatic data, land cover, desertification risk, aridity, soil type, normalized
vegetation index, bedrock geological formations. From the bacterial point of view, 
there curated databeses that classify in some baseline functionality. 

Disentagling the soil ecosystem functioning requires an holistic and 
multidisciplinary approach \cite{vogel2022}. The integration of the afformentioned
data is needed to understand the biogeochemical cycles along with biodiversity interactomes 
that have been characterised as a driver of community composition soil
functioning \cite{GUSEVA2022108604}.
Regarding the latter there are still big challenges to infer actual microbial
interactions remain \cite{Faust2021}. All this work is needed in order to meet
UN and EU soil goals for the 2030 and 2050 for healthy soils \cite{LAL2021e00398}.

In this study we ask: what the differences in the microbiome communities in different land use types?
Are there any climatic, geological, elevational, aridity or functional factors that affected the differences?
How the interactome changes over different land use types and climate?
Are there any distictions between arid regions of Crete?
To address these questions, we integrate multiple types of data and methods to decipher hidden 
signals of the Island Sampling Day soil microbiome data. In total, 144 samples from
72 sites (2 samples per site) of Crete are used with their metadata.
Warning signals are also identified for various types of ecosystems.


\section{Materials and Methods}\label{methods}

\begin{figure}[t] 
    \centering\includegraphics[width=\columnwidth]{crete_soil_microbiome}
    \caption{Workflow of this study. Data integration of ISD data with multiple types of spatial data. Then, a threefold analysis of function annotations, network analysis and differential abundance. All these data and methods are used to focus on specific taxa, ecosystems and threats.}
    \label{fig:workflow}
\end{figure}

\subsection{Island Sampling Day: Crete}\label{isd_data}

This study uses the ISD Crete data that have been deposited
in the European Nucleotide Archive (ENA) at EMBL-EBI under accession number PRJEB21776.
The raw sequences (fastq files) and metadata (xml files) were downloaded with custom scripts using the ENA API \cite{Yuan2023}.
Specific information regarding the ISD sampling, DNA extractions, PCR and sequencing
protocols are discussed here \cite{holm2024}.

Amplicon Sequence Variants were inferred using DADA2 \cite{Callahan2016} for 
filtering, denoising and chimeric reads removal. Normalisation of the reads
across samples was implemented with the SRS R package \cite{Beule2020}. Samples
with less than 10000 reads were removed. ASVs were assigned to taxonomy using 
DADA2 with Silva 138 \cite{Quast2012}.

\subsection{Crete data cube}\label{data}

The complilation of Crete data cube has multiple spatial data layers of global,
european, greek or cretan scale. 
The Copernicus CORINE Land Cover has 3 layers resolution that classify the land
use and cover in shapefile format \cite{CLC2023}. 
WorldClim 2.0 contains global climatic data for 12 variables, e.g annual mean
temperature and annual precipitation \cite{Fick2017}.
We also utilised teh Environmentally Sensitive Areas Index to desertification (ESAI) 
of Greece dataset \cite{KARAMESOUTI2018266}. Additionally we included the 
Global Aridity Index and Potential Evapotranspiration Database \cite{zomer2022version}.
Geological formations shapefiles were downloaded from the geoportal of
Decentralized Administration of Crete, which were developed by
Crinno-Emeric Group project\footnote{\url{https://geoportal.apdkritis.gov.gr/gis/apps/storymaps/stories/19690f65abbe4e8ab0141b2fe7261a8c}}.
Handling and analysis of these data was done with the sf and terra R packages \cite{Pebesma2023}.
%World Database on Protected Areas (WDPA) \cite{Hanson2022}
%Harmonized World Soil Database

\subsection{Integrative Analysis and Annotations}\label{int_analysis}
The network inferrence was facilitated with FlashWeave 0.19.2 \cite{Tackmann2019}.
To use FlashWeave we reduced 
the abandunce table of the ASVs to keep the ones at the genus or species level.
In addition, we filtered these taxa that appeared in SOSO samples and had more than 
SOSO mean relative abundance. The subsequent network analyses were carried out
with the igraph R package \cite{Csardi2006}.

For taxa function annotation we used the manually curated FAPROTAX database and python script \cite{Louca2016}.
Whereas for the differential abundance analysis with ANCOM-BC2 R package \cite{Lin2023}.
Numerical ecology analyses,e.g diversity indices, NMDS, PERMANOVA we calculated
with the vegan R package \cite{vegan}.
For PCoA ordination we used ape R packege \cite{Paradis2004} and UMAP python library\cite{mcinnes2018umap-software}.

\subsection{Tools}\label{Coding environment}
%PEMA for OTU inferrence \cite{Zafeiropoulos2020}
%U-CIE R package for coloring 3 dimentional data \cite{Koutrouli2022}

Visualisation was implemented with ggplot2 \cite{ggplot22016} and pheatmap \cite{Kolde2019}.
The environment we worked had Python 3.11.4, R version 4.3.2 \cite{rcoreteam}
and Julia language version 1.9.3 \cite{Julia-2017}in Julia language version 1.9.3 \cite{Julia-2017}.
Finally, computations were performed on HPC infrastrure of HCMR \cite{Zafeiropoulos2021}.

\section{Results}\label{results}

\subsection{Soil microbiome}\label{soil_microbiome}

Summary of the pipeline results

Generalists and specialists


\subsection{Crete Data Cube}\label{data_cube}



\subsection{Community drivers}\label{Drivers}

\subsection{Interactome}\label{interactome}

\subsection{Human interventions}\label{Human interventions}

\subsubsection{Functions}\label{functions}

\subsubsection{Significant taxa}\label{sig_taxa}

Differential abundance

\section{Discussion}\label{discussion}

Crete's ecosystems are mostly semi-arid, whereas in the mountain ranges there 
are areas classified as dry sub-humid and humid. 

Parent material and infuences on soil functions and bacterial communities.

Networks specialists generalists \cite{Barberan2012}

Elevation gradients of biodiversity are known since Humboldt's work \cite{Rahbek2019} 
yet these patterns remain elusive regarding the soil microbiome \cite{Looby2020, Siles2023}.
Mostly because it's difficult to isolate other cofounding effects \cite{Nottingham2018}. 

Functions of Richtis gorge are a signal in need to be further investigated.

Enumeratic all biodivarsity \cite{Anthony2023} and decipher the biogeochemical 
processes and interactions of fauna, flora and microbes \cite{Fry2019, Crowther2019,GRANDY201640,Delgado-Baquerizo2020}
Positive and negative in soils \cite{Liu2024}

Global hotspots \cite{Guerra2022}
and soil conservation instead of specific species \cite{Guerra2021}
and policy \cite{KONINGER2022} across countries \cite{Putten2023} and in Greece \cite{SCHISMENOS2022100035}

"A holistic perspective on soil architecture is needed as a key to soil functions" \cite{philippot2024the-interplay}
\section{Conclusion}

Mountain peaks soil microbes \cite{Adamczyk2019}
Even though amplicon studies in soil should be interpeted with caution \cite{alteio2021} they 
can act as early warning signals towards public health conserns \cite{Banerjee2023}.

The pillar of data integration is the unrestricted open data across disciplines and 
the open source software.
Deciphering and validating the results presented here requires future work.
Shotgun metagenomics and metatranscriptomics can uleash the functional potential of
topsoil along with other advancements like long reads sequencing. Higher resolution
samplings using grid system will enhance the resolution and also the resampling of
ISD sites in different time points will provide additional insights to the complex soil 
functions.



%%%%%%%%%%%%%%

\begin{appendices}

\section{Appendix 1}\label{appendix1}

Nam dui ligula, fringilla a, euismod sodales, sollicitudin vel, wisi. Morbi auctor lorem non justo. Nam lacus libero,
pretium at, lobortis vitae, ultricies et, tellus. Donec aliquet, tortor sed accumsan bibendum, erat ligula aliquet magna,
vitae ornare odio metus a mi. Morbi ac orci et nisl hendrerit mollis. Suspendisse ut massa. Cras nec ante. Pellentesque
a nulla. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Aliquam tincidunt
urna. Nulla ullamcorper vestibulum turpis. Pellentesque cursus luctus mauris.

\begin{table}[]
    \caption{Taxonomic depth of ASVs and the unique number of taxa of each level.\label{table:asv_taxonomy}}%
\begin{tabular}{@{}lll@{}}
classification depth & Total ASV    & Total taxa \\
Kingdom              & 1974         & 2          \\
Phylum               & 4034         & 33         \\
Order                & 38517        & 193        \\
Class                & 24157        & 83         \\
Family               & 71355        & 287        \\
Genus                & 90137        & 1166       \\
Species              & 9120         & 1338       \\
Total                & $\sim$239000 & 3102      
\end{tabular}
\label{table:asv_taxonomy}
\end{table}

%\section{Appendix 2}\label{appendix2}
%\subsubsection{Subsubsection title of first appendix}\label{subsubsec3}
%
%Fusce mauris. Vestibulum luctus nibh at lectus. Sed bibendum, nulla a faucibus semper, leo velit ultricies tellus, ac
%venenatis arcu wisi vel nisl. Vestibulum diam. Aliquam pellentesque, augue quis sagittis posuere, turpis lacus congue
%quam, in hendrerit risus eros eget felis.
%
\end{appendices}
\section{Data and Code}
The documentation and scripts developed for this study are available in
\href{https://github.com/savvas-paragkamian/crete_soil_microbiome/}{Crete soil microbiome github repository}.
This repository contains all the necessary scripts for the data retrieval,
filtering and ASV inferrence, taxonomy assignment, data integration of spatial data, 
functional annotation and the subsequent analyses and visualisation.
Code is structured to be reproducible and interoperable.

\section{Competing interests}
No competing interest is declared.

\section{Author contributions statement}
Conceptualization: LS, EP, GK, AM;
Data curation: JH, LS, EP, SY, SP;
Formal Analysis: SP, JH;
Funding acquisition: LS, AM, SP;
Investigation: LS, SY, SP, JH;
Methodology: SP, GK, EP, LS, SY, CC, CP, HZ, MS;
Project administration: LS, EP, GK;
Resources: EP, LS, SY, AM;
Software: SP, JH, HZ;
Supervision: LS, EP, GK, PS;
Validation: JH, CP, CC, HZ, PS, DT, MS, LS;
Visualization: SP, JH;
Writing – original draft: SP, EP, GK;
Writing – review and editing: All authors.

\section{Acknowledgments}
The authors thank the anonymous reviewers for their valuable suggestions.

\section{Funding}
SP was supported by the 3rd H.F.R.I. Scholarships for PHD Candidates (no. 5726) for his work.

\bibliographystyle{unsrt}
\bibliography{reference}


%USE THE BELOW OPTIONS IN CASE YOU NEED AUTHOR YEAR FORMAT.
%\bibliographystyle{abbrvnat}
%\bibliography{reference}


\end{document}
